cvat/apps/auto_segmentation/models.py

```diff
@@ -0,0 +1,9 @@
+
+# Copyright (C) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from django.db import models
```

Comment:
@MalekMFS , such unused imports lead to codacy errors. Could you please remove them?